Algorithm.CSharp/CustomDataIndicatorExtensionsAlgorithm.cs

```diff
@@ -0,0 +1,95 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System;
+using QuantConnect.Data;
+using QuantConnect.Data.Custom;
+using QuantConnect.Data.Market;
+using QuantConnect.Indicators;
+
+namespace QuantConnect.Algorithm.CSharp
+{
+    /// <summary>
+    /// The algorithm creates new indicator value with the existing indicator method by Indicator Extensions
+    /// Demonstration of using the external custom datasource Quandl to request the VIX and VXV daily data
+    /// </summary>
+    /// <meta name="tag" content="using data" />
+    /// <meta name="tag" content="using quantconnect" />
+    /// <meta name="tag" content="custom data" />
+    /// <meta name="tag" content="indicators" />
+    /// <meta name="tag" content="indicator classes" />
+    /// <meta name="tag" content="plotting indicators" />
+    /// <meta name="tag" content="charting" />
+    public class CustomDataIndicatorExtensionsAlgorithm : QCAlgorithm
+    {
+        private const string VIX = "CBOE/VIX";
+        private const string VXV = "CBOE/VXV";
+        private SimpleMovingAverage smaVIX;
+        private SimpleMovingAverage smaVXV;
+        private IndicatorBase<IndicatorDataPoint> ratio;
+
+        /// <summary>
+        /// Initialize the data and resolution you require for your strategy
+        /// </summary>
+        public override void Initialize()
+        {
+            SetStartDate(2014, 1, 1);
+            SetEndDate(2018, 1, 1);
+            SetCash(25000);
+
+            // Define the symbol and "type" of our generic data
+            AddData<QuandlVix>(VIX, Resolution.Daily);
+            AddData<Quandl>(VXV, Resolution.Daily);
+            // Set up default Indicators, these are just 'identities' of the closing price
+            smaVIX = SMA(VIX, 1);
+            smaVXV = SMA(VXV, 1);
+            // This will create a new indicator whose value is smaVXV / smaVIX
+            ratio = smaVXV.Over(smaVIX);
+        }
+
+        /// <summary>
+        /// Custom data event handler:
+        /// </summary>
+        /// <param name="data">Quandl - dictionary Bars of Quandl Data</param>
+        public void OnData(Quandl data)
+        {
+            // Wait for all indicators to fully initialize
+            if (smaVIX.IsReady && smaVXV.IsReady && ratio.IsReady)
+            {
+                if (!Portfolio.Invested && ratio > 1)
+                {
+                    MarketOrder(VIX, 100);
+                }
+                else if (ratio < 1)
+                {
+                    Liquidate();
+                }
+                // plot our indicators 
+                Plot("SMA", smaVIX, smaVXV);
+                Plot("Ratio", ratio);
```

Comment:
In the python example, you used `PlotIndicator`, should both algorithms use the same methods for consistency?